.jenkins

```diff
@@ -142,7 +142,7 @@ pipeline {
                         dockerfile {
                             filename "Dockerfile"
                             dir "docker"
-                            additionalBuildArgs '--build-arg BASE=ubuntu:18.04 --build-arg KOKKOS_OPTIONS="--with-serial --with-openmp --arch=AMD --compiler=clang++ --cxxstandard=c++14"'
+                            additionalBuildArgs '--build-arg BASE=ubuntu:18.04 --build-arg KOKKOS_OPTIONS="-DCMAKE_CXX_STANDARD=14 -DKokkos_ENABLE_SERIAL=ON -DKokkos_ENABLE_OPENMP=ON -DCMAKE_CXX_COMPILER=clang++ -DKokkos_ARCH_AMDAVX=ON"'
```

Comment:
`-DKokkos_ARCH_AMDAVX=ON` is not safe no that we have more slave machines with the same "docker" label

Reply:
I don't see a problem with just dropping it.

Reply:
Done